misc/e2elive.py

```diff
@@ -57,7 +57,7 @@ def main():
         from botocore.config import Config
         from botocore import UNSIGNED
         s3 = boto3.client('s3', config=Config(signature_version=UNSIGNED))
-        tarname = 'net_done.tar.bz2'
+        tarname = 'feature-stateproofs.tar.bz2'
```

Comment:
stateproofs is merged! We can change this:
```suggestion
        tarname = 'rel-nightly.tar.bz2'
```